pkg/qe-tests/cypress/integration/models/plan.ts

```diff
@@ -440,7 +440,6 @@ export class Plan {
     applyAction(name, restartButton);
     confirm(); //Added Confirm Button
     if (warmMigration) {
-      Plan.openList();
```

Comment:
You don't have to open the plan list anymore ?

Reply:
I guess it was left here by mistake, it is not harmful but just not needed here, I talked with Karishma about this on previous review.

Reply:
This is related to this merged PR https://github.com/konveyor/forklift-ui/pull/913. It was my mistate I missed to remove the " Plan.openList();" in previous PR. Previously did some modification to the code and hence covered inside it :)